catalog/MDCCatalog/AppDelegate.swift

```diff
@@ -113,5 +115,12 @@ extension UINavigationController {
     }
     return container
   }
+  class func getScrollView(vc: UIViewController) -> UIScrollView? {
+    let scrollViewSel = NSSelectorFromString("scrollView");
+    if vc.responds(to: scrollViewSel) {
+      return vc.perform(scrollViewSel).takeRetainedValue() as? UIScrollView
+    }
+    return nil
```

Comment:
A recursive traversal for a UIScrollView will scale for other examples better. Consider doing that instead of runtime inspection.

Reply:
what about UICollectionViews in a UICollectionView rather than a UICollectionViewController.